### PR TITLE
Feature - Primary Address for Personal Account Details

### PR DESCRIPTION
**Pull request by ztrouy**
### Purpose
To provide the Primary Address of the logged in User when fetching their personal account details.

### Files Changed
- Added related_name of "active_address" to the User property in `user_address.py`
- Added the `primary_address` calculated property that returns the Users' Address set to be their primary Address, if on exists in `user.py`
- Updated get_my_details method to include the Users' primary Address when possible in `user.py`

### To Test
Fetch, run the following commands in the project root directory
```
pipenv shell
```
```
pipenv install
```
```
./seed_database.sh
```
Then confirm the following
- GET endpoint `/users/me` returns the account details of the logged in User, including primary Address is exists

### CODE REVIEW

**Review by ezjbrewer — APPROVED** [review]
Beautiful work, Lord Overseer!

File: gondorapi/models/user.py [hunk 1]
```diff
@@ -18,5 +19,18 @@ class User(AbstractUser):
...
+    @property
+    def primary_address(self):
+        try:
+            primary_address_type = AddressType.objects.get(name="Primary Address")
+            user_address = self.active_addresses.get(address_type=primary_address_type)
+
+            if user_address:
+                return user_address.address
+            
+            return None
```
Comment (ccote95) [inline gondorapi/models/user.py:31]: WHITE SPACE

**Review by ztrouy — COMMENTED** [review]
All good

**Comment by ccote95** [comment]
:shipit:

**Review by ccote95 — APPROVED** [review]
:shipit:

**Pull request merged**